test/CMakeLists.txt

```diff
@@ -161,11 +161,11 @@ target_compile_definitions(ArborX_Clustering.exe PRIVATE BOOST_TEST_DYN_LINK)
 target_include_directories(ArborX_Clustering.exe PRIVATE ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_SOURCE_DIR}/examples/dbscan)
 add_test(NAME ArborX_Clustering_Test COMMAND ./ArborX_Clustering.exe)
 
-add_executable(ArborX_TreeHelpers.exe tstTreeHelpers.cpp utf_main.cpp)
-target_link_libraries(ArborX_TreeHelpers.exe PRIVATE ArborX Boost::unit_test_framework)
-target_compile_definitions(ArborX_TreeHelpers.exe PRIVATE BOOST_TEST_DYN_LINK)
-target_include_directories(ArborX_TreeHelpers.exe PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
-add_test(NAME ArborX_TreeHelpers_Test COMMAND ./ArborX_TreeHelpers.exe)
+add_executable(ArborX_TreeNodeLabelling.exe tstDetailsTreeNodeLabelling.cpp utf_main.cpp)
```

Comment:
In the American English it's `labeling`, while in UK it's `labelling`. I would prefer the single `l`, but not a blocker.

Reply:
OK with updating, didn't realize.